404-server/Godeps/Godeps.json

```diff
@@ -1,8 +1,15 @@
 {
 	"ImportPath": "k8s.io/contrib/404-server",
-	"GoVersion": "go1.5",
+	"GoVersion": "go1.7",
+	"GodepVersion": "v78",
 	"Packages": [
-		"./..."
+		"./.."
```

Comment:
You've made a mistake here. You should type:
```
godep save ./...
```
but you've typed
```
godep save ./..
```
The different is just one dot. The should be 3. Something weird had happened. Please try again.